HCaptcha/Classes/Journeylitics/FieldKey.swift

```diff
@@ -0,0 +1,63 @@
+import Foundation
+
+/**
+ * Serialization enum for consistent field mapping across platforms
+ * Maps readable field names to short JSON keys for minification
+ */
+public enum FieldKey: String {
+    // Top-level fields (always present)
+    case kind = "k"
+    case view = "v"
+    case timestamp = "ts"
+    case meta = "m"
+
+    // Meta fields (nested under meta object)
+    case id = "i"
+    case screen = "s"
+    case action = "a"
+    case value = "val"
+    case x = "x"
+    case y = "y"
+    case index = "idx"
+    case section = "sct"
```

Comment:
```suggestion
    case section = "sect"
```

mismatch to above ?

Reply:
It's actually short name we can keep any of it